notesapplication/notes/api_v1/serializers.py

```diff
@@ -40,36 +76,56 @@ def update(self, instance, validated_data):
         )
         return super().update(instance, validated_data)
 
+    def to_representation(self, instance):
```

Comment:
don't you think this code block could be improved in terms of readability and maintainability? Maybe just show the last comment like `note.comments.last()` and add that in the repr.